common/macrosLeeper.tex

```diff
@@ -271,8 +274,10 @@
 %%%
 \newcommand{\rotationTable}[2]
 {
-  \def\dirCosTableA{#1}
-  \def\dirCosTableB{#2}
+  \def\dirCosTablea{\lowercase{#1}}
+  \def\dirCosTablea{\lowercase{#2}}
+  \def\dirCosTableA{\uppercase{#1}}
+  \def\dirCosTableB{\uppercase{#2}}
```

Comment:
I see what you were trying to fix. I thought about it and I think this is the right solution.